brownie_tokens/forked.py

```diff
@@ -34,6 +36,16 @@ def _mint_for_testing(self, target: str, amount: Wei, tx: Dict = None) -> None:
             getattr(sys.modules[__name__], fn_name)(self, target, amount)
             return
 
+        # check for token name if no custom minting
+        # logic exists for address
+        for name in _token_names:
+            if hasattr(self, 'name') and self.name().startswith(name):
+                fn_name = f"mint_{name}"
+                if hasattr(sys.modules[__name__], fn_name):
+                    getattr(sys.modules[__name__], fn_name)(
+                        MintableForkToken(self.UNDERLYING_ASSET_ADDRESS()), target, amount)
```

Comment:
`UNDERLYING_ASSET_ADDRESS` is very specific to Aave I think? So this logic will work right now, but could cause issues if we later expand to handle other tokens based on now the name starts.

Reply:
Yes, that should definitely be handled only in the case of Aave. Will refactor!